.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "Debug APIServer",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/cmd/apiserver/apiserver.go"
+        },
+        {
+            "name": "Debug Controller Manager",
+            "type": "go",
+            "request": "launch",
+            "mode": "auto",
+            "program": "${workspaceFolder}/cmd/controller/main.go"
+        }
+    ]
+}
```

Comment:
Please don't submit a particular IDE settings file. And put it into `.gitignore`

Reply:
When we use [Gitpod](https://www.gitpod.io/) or [Codespace](https://github.com/features/codespaces) to debug code online or debug locally with [Visual Studio Code](https://code.visualstudio.com/), it's very meaningful. I apply to keep them.

Reply:
What happened if there are over 10 IDEs? Different contributors will submit different config files. It's terrible. I can understand it's helpful for you, but not for me.